app/cdap/components/WranglerGrid/SelectColumnPanel/constants.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IMultipleSelectedFunctionDetail } from 'components/WranglerGrid/SelectColumnPanel/types';
+
+export const MULTI_SELECTION_COLUMN: IMultipleSelectedFunctionDetail[] = [
+  {
+    value: 'join-columns',
+    isMoreThanTwo: false,
```

Comment:
What does isMoreThanTwo mean here? So this is a multi select column list and certain ones can have more than two selection values?

Reply:
Yes, join and swap column requires two column to be selected not more than two while others like delete, keep can have more than two selections